source/documentation/runbooks/dns/ns-changes-involving-gds.html.md.erb

```diff
@@ -0,0 +1,55 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Changes to Nameserver Records involving GDS Domains Team
+last_reviewed_on: 2023-05-22
+review_in: 3 months
+---
+
+# Changes to Nameserver Records involving GDS Domains Team
+
+This runbook is created to help with planning and implementation of Nameserver record changes that will involve the GDS Domains Team. This would normally occur when we are migrating a service from one hosting platform to another.
+
+This will typically be service that have domain names in the format `example.service.gov.uk`, `example.gov.uk` or `example.independant-inquiry.gov.uk`.
+
+## Check if this change will involve BT or direct engagement with the GDS Domains Team.
+
+There are two slightly different processes depending on where we manage our domains. If the domain is present in BT-DNS then this process will involve BT, otherwise change requests will need to be made to the GDS Domains Team.
+
+## Scheduling Changes
+
+When scheduling cutovers with Service Teams you should ensure that teams are aware of a number of things:
+* Both GDS and BT have a 5 day SLA for change requests
+* When nameserver changes are made there could a 6 hour caching delay whilst the new nameservers take over management of DNS
+* Change involving BT **cannot** take place out of hours, and can only be implemeneted at two specific times of the day
```

Comment:
What are the two specific times of the day? I'm on tenterhooks :)

Reply:
Added times.